ngraph/frontend/paddlepaddle/src/op/top_k.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ngraph/opsets/opset6.hpp>
+#include <node_context.hpp>
+
+namespace ov {
+namespace frontend {
+namespace pdpd {
+namespace op {
+NamedOutputs top_k(const NodeContext& node) {
```

Comment:
Can we merge it with topk_v2? topk is a special case of topk_v2. We'd better take care of them together to reuse code.